includes/class-wc-calypso-bridge-setup.php

```diff
@@ -273,6 +286,7 @@ public function woocommerce_create_pages_callback() {
 					$page = get_page_by_path( $page, ARRAY_A );
 					if ( is_array( $page ) && isset( $page['ID'] ) ) {
 						wp_delete_post( $page['ID'], true );
+						$this->write_to_log( $operation, 'deleted WooCommerce page ' . $page );
```

Comment:
The `$pages` is probably an array. We should use the page name

Reply:
This `$page` is not an array, it's in a loop of an array

Reply:
Scratch that, good catch